packages/internal/src/ast.ts

```diff
@@ -5,12 +5,13 @@ import { types } from '@babel/core'
 import type { ParserPlugin } from '@babel/parser'
 import { parse as babelParse } from '@babel/parser'
 import traverse from '@babel/traverse'
-import chalk from 'chalk'
 
 import { getPaths } from '@cedarjs/project-config'
 
 import { isFileInsideFolder } from './files'
 
+const ansis = require('ansis')
```

Comment:
Since the `internal` is `CommonJS` package and used strictly checking with the `"moduleResolution": "Node16"` or `"NodeNext"` option in tsconfig, using `import` in CJS mode is impossible.

Ansis is DUAL package, support both ESM and CJS. In strict CJS mode TS treats Ansis as pure `CommonJS` package, therefore importing is not allowed.

More info and possible solutions you can see in [Troubleshooting](https://github.com/cedarjs/cedar/pull/253/files#troubleshooting).